### ISSUE TITLE
"Manual" settings for color/size/symbol

### ISSUE THREAD

**Issue by ceriottm** [cosmo-epfl/chemiscope#148]
ATM when color/size/symbol are not linked to a property, they are set to a default value. 
Would be nice to have a "manual" setting to allow choosing them, well, manually.

**Comment by ceriottm** [comment]
Mentioned in PR #144

**Comment by Luthaf** [comment]
Isn't this fixed with the new settings infrastructure?

**Comment by ceriottm** [comment]
> Isn't this fixed with the new settings infrastructure?

No, this was meant to allow choosing an actual color - at present it picks the midpoint on the chosen palette. and the size can be actually set so probably we can close until someone comes with a pressing need to have crosses rather than circles as default
